apps/fz_http/priv/repo/migrations/20221201216913_create_gateway_firewall_policies.exs

```diff
@@ -0,0 +1,68 @@
+defmodule FzHttp.Repo.Migrations.CreateGatewayFirewallPolicies do
```

Comment:
I think we should consider using two tables here, `network_policies` and a join table `gateway_network_policies` to support managing policies as first-class citizens in Firezone. What do you think?

The `network_policies` would then own the `default_action` field, and we could limit the user to attaching one network policy to a gateway for now.

This would allow policies to be applied to potentially many gateways, which I assume would be a common usage pattern.

Reply:
Yes, I think that's a good idea. Wouldn't the `network_policies` table also own the other fields like port and destination, etc?